packages/common/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@web3modal/common",
+  "version": "3.2.0",
+  "type": "module",
+  "main": "./dist/esm/index.js",
+  "types": "./dist/types/index.d.ts",
+  "files": [
+    "dist",
+    "!tsconfig.tsbuildinfo"
+  ],
+  "scripts": {
+    "build:clean": "rm -rf dist",
+    "build": "npm run build:clean; tsc --build",
+    "watch": "npm run build:clean; tsc --watch",
+    "typecheck": "tsc --noEmit",
+    "test": "vitest run --dir tests",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+  },
+  "dependencies": {
+    "dayjs": "1.11.10"
+  },
+  "keywords": [
+    "web3",
```

Comment:
Adjust keywords to match the package a bit more i.e. lit and webcomponents are not needed here probably

Reply:
Added `utils`, `types`, `web3`, `web3modal`